Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/WebChatLog/WebChatLogContent.tsx

```diff
@@ -160,14 +179,39 @@ export const WebChatLogContent: React.FC<DebugPanelTabHeaderProps> = ({ isActive
     }
   };
 
+  const onOpenWebChatPanelClick = () => {
+    setWebChatPanelVisibility(true);
+  };
+
+  const noWebChatTrafficSection = useMemo(() => {
+    if (currentStatus === BotStatus.inactive) {
+      return (
+        <div css={emptyStateMessageContainer}>
```

Comment:
nit: highly recommend using formatMessage.rich for this and the following on L200